source/Nevermore/RelationalJsonContractResolver.cs

```diff
@@ -16,9 +16,37 @@ public RelationalJsonContractResolver(IRelationalStoreConfiguration configuratio
             this.configuration = configuration;
         }
 
-        protected override JsonProperty CreateProperty(MemberInfo member, MemberSerialization memberSerialization)
+        protected override IList<JsonProperty> CreateProperties(Type type, MemberSerialization memberSerialization)
         {
-            configuration.DocumentMaps.ResolveOptional(member.DeclaringType, out var map);
+            var members = GetSerializableMembers(type);
+            if (members == null)
+            {
+                throw new JsonSerializationException("Null collection of serializable members returned.");
+            }
+
+            var properties = new JsonPropertyCollection(type);
+
+            foreach (var member in members)
+            {
+                var property = CreateProperty(type, member, memberSerialization);
+
+                if (property != null)
+                {
+                    properties.AddProperty(property);
+                }
+            }
+
+            var orderedProperties = properties.OrderBy(p => p.Order ?? -1).ToList();
+            return orderedProperties;
```

Comment:
This is mostly copied from the base class with the exception of the `DefaultJsonNameTable`, since that's an internal field. It seems like it's used as a cache, and the tests are all passing with this change, so I'm fairly confident this is solid 👍 

https://github.com/JamesNK/Newtonsoft.Json/blob/f7e7bd05d9280f17993500085202ff4ea150564a/Src/Newtonsoft.Json/Serialization/DefaultContractResolver.cs#L1362-L1390